index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html><html>
+
+  <head> <title>YouTube</title>
+      <link rel="stylesheet" href="style.css" type="text/css">
+  </head>
+  <body>
+      <div class="nav-bar">
```

Comment:
[OPTIONAL]
It's best practice to use the `nav` tag for the navbar. This will make your code more semantic and by doing that it will improve the quality of it.